examples/ola-recorder.cpp

```diff
@@ -112,6 +112,21 @@ int RecordShow() {
 }
 
 
+/**
+ * Clamps the frame count between 0 and 1.
+ *
+ * This allows frames that would be cached during playback to be counted.
+ */
+void ClampVerifyFrameCount(map<unsigned int, unsigned int> &frames) {
+  map<unsigned int, unsigned int>::iterator iter;
+  for (iter = frames.begin(); iter != frames.end(); ++iter) {
+    if (iter->second > 1) {
+      iter->second = 1;
+    }
+  }
+}
```

Comment:
Does this actually need to be a whole function, it only seems to get used once in one place, wouldn't a simple loop through the frames when we go into playback mode be just as easy (without needing to pass the frame counter in etc?

Reply:
Not really, just trying to stop VerifyShow from becoming any more unwieldy.